staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go

```diff
@@ -177,8 +183,9 @@ func (f *ConfigFlags) ToRawKubeConfigLoader() clientcmd.ClientConfig {
 	} else {
 		clientConfig = clientcmd.NewInteractiveDeferredLoadingClientConfig(loadingRules, overrides, os.Stdin)
 	}
+	f.clientConfig = clientConfig
```

Comment:
is this threadsafe?

Reply:
That's a massive change to the contract - we really can't do that.

Reply:
an acceptable alternative could be a second method that does something similar to this (though in a threadsafe manner) with a name that makes it clear the returned value is reusing a previously computed value if available